### PR TITLE
Disable shadows in secondary camera

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.manuscript.com/f/cases/15957/Shadows-don-t-render-correctly-in-the-Secondary-Camera

Test plan:
- Go somewhere with a mirror and shadows.  dev-mobile has a mirror on the cart near the starting area, and has shadows on.  You should see shadows in the main view, but not in the mirror.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10900/HighFidelity-Beta-PR13468-97ee028.apk

File: interface/src/SecondaryCamera.h [hunk 1]
```diff
@@ -12,23 +12,11 @@
...
 
-class MainRenderTask {
-public:
-    using JobModel = render::Task::Model<MainRenderTask>;
-
-    MainRenderTask() {}
-
-    void build(JobModel& task, const render::Varying& inputs, render::Varying& outputs, render::CullFunctor cullFunctor, bool isDeferred = true);
-};
-
```
Comment (AndrewMeadows) [inline interface/src/SecondaryCamera.h]: This and all above changes are cruft removal and cleanup, not related to the actual fix, right?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10900/HighFidelity-Beta-PR13468-97ee028.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10900/HighFidelity-Beta-Interface-PR13468-97ee028.dmg

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Shadows do not appear in mirrors

**Pull request merged**